tic-tac-toe/src/components/info-block/info-block.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './info-block.css';
+
+const InfoBlock = ({info, outerClassName}) => {
+    const innerClassName = 'info-block';
+    const className = `${outerClassName} ${innerClassName}`
```

Comment:
It's overkill here. Just add a className to your component.

Reply:
I fixed this.
```suggestion
const InfoBlock = ({info}) => {
  return (
    <div className={classes.Block}>
      {info}
    </div>
  )
}
```